x-pack/plugin/monitoring/src/test/java/org/elasticsearch/xpack/monitoring/integration/MonitoringIT.java

```diff
@@ -374,9 +374,6 @@ private void assertClusterStatsMonitoringDoc(final Map<String, Object> document,
         assertThat(clusterState.remove("cluster_uuid"), notNullValue());
         assertThat(clusterState.remove("master_node"), notNullValue());
         assertThat(clusterState.remove("nodes"), notNullValue());
```

Comment:
Curious to why we are removing these assertions ?

Reply:
I think it's been added for Zen 2 but removed afterwards, but not noticed since the test was muted.